acceptance_tests/Gemfile

```diff
@@ -0,0 +1,7 @@
+# A sample Gemfile
+source "https://rubygems.org"
+
+gem cucumber
+gem capybara
+gem poltergeist
+gem rspec
```

Comment:
I might be wrong, but shouldn't these gem values be in single quotes and followed by version numbers?